Support for multiple DLL export functions parsed from a DLL file or comma separated

mariolima opened the pull request:
## Possible uses

`./Freeze -I loader.bin -O loader.dll -export SetLnkAUMI,UninstAppUserModelId,UninstShortcut`
Will create a loader with the exported the functions `SetLnkAUMI`,`UninstAppUserModelId`,`UninstShortcut`

`./Freeze -I loader.bin -O loader.dll -export wer.dll`
Will parse the given `wer.dll` DLL with [Binject/debug/pe](https://github.com/Binject/debug) and use its Export functions in the loader.

Feel free to ignore this PR if it doesn't fit your view of the project btw! :) I just found this super useful for DLL sideloading.

mariolima committed:
ability to use a DLL export list cloned from a given DLL file

mariolima committed:
ability to provide multiple comma separated Export functions

Tylous commented:
I will take a look at this and get back to you.

Tylous commented:
This doesn't give multiple exports, it actually gives one long. I need to tweak the struct. to allow various different function names. Will adjust in the new version.

mariolima committed:
fixed issue with duplicated exports and removal of the default ones
- exported functions are now all passed in an array
- exports array defaults to DllRegisterServer, DllGetClassObject, DllUnregisterServer
- ExportsFromFile is not in utils.go

mariolima commented:
Ah I see. Thanks for looking into this!

I believe I got it working with all variations:
```./Freeze -I loader.bin -O loader.dll -export "SetLnkAUMI,Testing"```

![image](https://user-images.githubusercontent.com/5732425/217611062-f58c75a5-18da-491f-a97e-e40a6146f60e.png)

```./Freeze -I loader.bin -O loader.dll -export wdscore.dll```
![image](https://user-images.githubusercontent.com/5732425/217611318-c369967f-7d25-4b34-a226-fe9c727a32fa.png)

```./Freeze -I loader.bin -O loader.dll```
![image](https://user-images.githubusercontent.com/5732425/217611462-4efe9b45-da39-43f3-bf25-80b5158ea91f.png)

mariolima committed:
Merge branch 'main' into main
